tests/ios/unit-test/test-imports-app/BUILD.bazel

```diff
@@ -76,5 +76,10 @@ ios_unit_test(
     visibility = ["//visibility:public"],
     deps = [
         ":TestImports-App_framework_unlinked",
-    ],
+    ] + select({
+        # Note: without using apple_framework, it won't set enable_framework_vfs
+        # imported it won't actually be seen
```

Comment:
hard to understand what this line means... `who` imported `what` won't actually be seen?

Reply:
Ah good point, I think this one is out of place.